storage/postgres/abstract.go

```diff
@@ -86,21 +89,100 @@ func create(ctx context.Context, db pgDB, table string, dto interface{}) (string
 		err = stmt.GetContext(ctx, &lastInsertId, dto)
 		return lastInsertId, checkIntegrityViolation(ctx, checkUniqueViolation(ctx, err))
 	}
-	log.C(ctx).Debugf("Executing query %s", query)
-	_, err := db.NamedExecContext(ctx, query, dto)
-
+	log.C(ctx).Debugf("Executing query %s", sqlQuery)
+	_, err := db.NamedExecContext(ctx, sqlQuery, dto)
 	return lastInsertId, checkIntegrityViolation(ctx, checkUniqueViolation(ctx, err))
 }
 
 func get(ctx context.Context, db getterContext, id string, table string, dto interface{}) error {
-	query := "SELECT * FROM " + table + " WHERE id=$1"
-	log.C(ctx).Debugf("Executing query %s", query)
-	err := db.GetContext(ctx, dto, query, &id)
+	sqlQuery := "SELECT * FROM " + table + " WHERE id=$1"
+	log.C(ctx).Debugf("Executing query %s", sqlQuery)
+	err := db.GetContext(ctx, dto, sqlQuery, &id)
 	return checkSQLNoRows(err)
 }
 
+func listWithLabelsByCriteria(ctx context.Context, db pgDB, baseEntity interface{}, labelsEntity Labelable, baseTableName string, criteria []query.Criterion) (*sqlx.Rows, error) {
+	if err := validateFieldQueryParams(baseEntity, criteria); err != nil {
+		return nil, err
+	}
+	var baseQuery string
+	if labelsEntity == nil {
+		baseQuery = constructBaseQueryForEntity(baseTableName)
+	} else {
+		baseQuery = constructBaseQueryForLabelable(labelsEntity, baseTableName)
+	}
+	sqlQuery, queryParams, err := buildQueryWithParams(db, baseQuery, baseTableName, labelsEntity, criteria)
+	if err != nil {
+		return nil, err
+	}
+	log.C(ctx).Debugf("Executing query %s", sqlQuery)
```

Comment:
Log the query and the parameters as well

Reply:
Not sure if it's okay to log the parameters